product/models.py

```diff
@@ -0,0 +1,98 @@
+from django.db import models
+
+class Category(models.Model):
+    name = models.CharField(max_length=50)
+
+    class Meta:
+        db_table = 'categories'
+
+class SubCategory(models.Model):
+    name     = models.CharField(max_length=50)
+    category = models.ForeignKey(Category, on_delete = models.SET_NULL, null = True)
+
+    class Meta:
+        db_table = 'sub_categories'
+
+class Product(models.Model):
+    name               = models.CharField(max_length=50)
+    name_en            = models.CharField(max_length=50)
+    image_url          = models.CharField(max_length=2000)
+    description        = models.TextField(null=True)
+    default_price      = models.IntegerField(null = True)
```

Comment:
```suggestion
    default_price      = models.DecimalField(null = True)
```
가격 필드는 'integer' 보다 'decimal'을 주로 사용합니다. 소숫점 둘째짜리까지 허용되도록 필드 수정해주세요